docs/HOWTO_AZURE_DEPLOY.md

```diff
@@ -0,0 +1,149 @@
+# Azure Deployment of Spoke
+
+## SERVER SET UP
+- In the Azure portal, create a resource group.
+- Set up an Azure account
+- Click start free
+- Sign up with Github!
+- Fill out information
+- They will ask for CC information, but don’t worry, they won’t charge for the first $200, so don’t spend up a huge rendering farm or something.
+- Click the Azure Portal
+- We will hopefully be able to incorporate a template for this section so new users don’t have to go through this lengthy process
+
+## Processor, memory, disk
+- Pick a virtual machine that Microsoft gives you (with Public IP Address and whatever amount of storage you selected to be associated with the machine.)
+- Choose Linux -- Ubuntu 20.04 (the current long term support release; Ubuntu is released in two year LTS cycles (next will be 22.04))
+- Create new resource group
+
+### For Size 
+- Click ‘see all sizes’
+- Azure Virtual Machine Selector (how to choose which one virtual machine is good for your needs.)
+- FOR OUR PURPOSES: Click on ‘B-series’ (cheaper basic version, users should upgrade for production deployment - potentially something in the D-series, e.g. D2v3, 2 production virtual CPUs, 8 GB of memory, 50 GB of temporary storage)
+- B1s
+- General purpose
+- 1 virtual CPU
+- 1 GB of memory
+- 4 GB of temporary storage
+- 320
+- Click Select
+- Verify you picked the right one: Size says Standard_B1s - 1vcpu, 1GiB memory ($7.59/month)
+-	Depending on how you create it, you may be prompted for “Next: Disks >” which will allow you to attach permanent storage
+
+### Create the Username and Password as primary administrator
+- Set up with Password
+- Inbound Port Rules, leave as is. 
+- Keeping port 22 open will allow us to SSH to clone Spoke onto our new server. (You may be warned: You have set SSH port(s) open to the internet.  This is only recommended for testing.  If you want to change this setting, go back to Basics tab.)
+
+### Enter your virtual machine,
+- Click Review + create
+- Create
+- Click Go to resource
+- We want to ssh into the new server
+- Microsoft will provide you with a public IP address (which is static)
+- In terminal: $ssh <your-username>@<static IP address>
+- Enter your password for this user
+- Note for later(?) for production purposes, you will want to attach permanent disk storage
```

Comment:
What does `Note for later(?)` mean?

Reply:
I flushed out the thought more, here are my changes:

- For production purposes, you will want to attach permanent disk storage (Documentation Update Coming Soon)